BENCH_DATA M4_COMP_BUSINESS-INDUSTRY <Bench.TS_datasets.cTimeSeriesDatasetSpec object at 0x7f3f69a00320>
TIME :  Date N= 12 H= 12 HEAD= [0 1 2 3 4] TAIL= [ 7  8  9 10 11]
SIGNAL :  B&IND0817 N= 12 H= 12 HEAD= [  10.  593.  827.  292.  587.] TAIL= [ 1059.  1036.   489.   595.   272.]
   B&IND0817  Date
0       10.0     0
1      593.0     1
2      827.0     2
3      292.0     3
4      587.0     4
<class 'pandas.core.frame.DataFrame'>
Int64Index: 12 entries, 0 to 11
Data columns (total 2 columns):
B&IND0817    12 non-null float64
Date         12 non-null int64
dtypes: float64(1), int64(1)
memory usage: 288.0 bytes
None
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M4Comp.py", line 17, in <module>
    tester.run_multiprocessed();
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(nbprocesses)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 140, in trainModel
    lAutoF1.train(df , self.mTSSpec.mTimeVar , iSignal, iHorizon)
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 25, in train
    self.mSignalDecomposition.train(iInputDS, iTime, iSignal, iHorizon, iExogenousData);
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 459, in train
    logger.info("START_TRAINING '" + iSignal + "'")
Message: "START_TRAINING 'B&IND0817'"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M4Comp.py", line 17, in <module>
    tester.run_multiprocessed();
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(nbprocesses)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 140, in trainModel
    lAutoF1.train(df , self.mTSSpec.mTimeVar , iSignal, iHorizon)
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 25, in train
    self.mSignalDecomposition.train(iInputDS, iTime, iSignal, iHorizon, iExogenousData);
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 481, in train
    logger.info("END_TRAINING_TIME_IN_SECONDS '" + iSignal + "' " + str(self.mTrainingTime))
Message: "END_TRAINING_TIME_IN_SECONDS 'B&IND0817' 4.775904417037964"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M4Comp.py", line 17, in <module>
    tester.run_multiprocessed();
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(nbprocesses)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 85, in getInfo
    logger.info("TIME_DETAIL " + self.mTrend.mTimeInfo.info());
Message: "TIME_DETAIL TimeVariable='Date' TimeMin=0 TimeMax=11 TimeDelta=1.0 Estimation = (0 , 12) Validation = (0 , 12) Test = (0 , 12) Horizon=12"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M4Comp.py", line 17, in <module>
    tester.run_multiprocessed();
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(nbprocesses)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 86, in getInfo
    logger.info("SIGNAL_DETAIL " + self.info());
Message: "SIGNAL_DETAIL SignalVariable='_B&IND0817' Min=10.0 Max=1059.0  Mean=580.333333333 StdDev=303.686607468"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M4Comp.py", line 17, in <module>
    tester.run_multiprocessed();
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(nbprocesses)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 87, in getInfo
    logger.info("BEST_TRANSOFORMATION_TYPE '" + self.mTransformation.get_name("") + "'");
Message: "BEST_TRANSOFORMATION_TYPE '_'"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M4Comp.py", line 17, in <module>
    tester.run_multiprocessed();
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(nbprocesses)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 88, in getInfo
    logger.info("BEST_DECOMPOSITION  '" + self.mOutName + "' [" + self.getFormula() + "]");
Message: "BEST_DECOMPOSITION  '_B&IND0817_Lag1Trend_residue_zeroCycle_residue_NoAR' [Lag1Trend + NoCycle + NoAR]"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M4Comp.py", line 17, in <module>
    tester.run_multiprocessed();
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(nbprocesses)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 89, in getInfo
    logger.info("TREND_DETAIL '" + self.mTrend.mOutName + "' [" + self.mTrend.mFormula + "]");
Message: "TREND_DETAIL '_B&IND0817_Lag1Trend' [Lag1Trend]"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M4Comp.py", line 17, in <module>
    tester.run_multiprocessed();
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(nbprocesses)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 90, in getInfo
    logger.info("CYCLE_DETAIL '"+ self.mCycle.mOutName + "' [" + self.mCycle.mFormula + "]");
Message: "CYCLE_DETAIL '_B&IND0817_Lag1Trend_residue_zeroCycle' [NoCycle]"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M4Comp.py", line 17, in <module>
    tester.run_multiprocessed();
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(nbprocesses)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 91, in getInfo
    logger.info("AUTOREG_DETAIL '" + self.mAR.mOutName + "' [" + self.mAR.mFormula + "]");
Message: "AUTOREG_DETAIL '_B&IND0817_Lag1Trend_residue_zeroCycle_residue_NoAR' [NoAR]"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M4Comp.py", line 17, in <module>
    tester.run_multiprocessed();
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(nbprocesses)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 92, in getInfo
    logger.info("MODEL_MAPE MAPE_Fit=" + str(self.mFitPerf.mMAPE) + " MAPE_Forecast=" + str(self.mForecastPerf.mMAPE)  + " MAPE_Test=" + str(self.mTestPerf.mMAPE) );
Message: 'MODEL_MAPE MAPE_Fit=0.6812 MAPE_Forecast=0.6812 MAPE_Test=0.6812'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M4Comp.py", line 17, in <module>
    tester.run_multiprocessed();
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(nbprocesses)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 93, in getInfo
    logger.info("MODEL_MASE MASE_Fit=" + str(self.mFitPerf.mMASE) + " MASE_Forecast=" + str(self.mForecastPerf.mMASE)  + " MASE_Test=" + str(self.mTestPerf.mMASE) );
Message: 'MODEL_MASE MASE_Fit=0.9167 MASE_Forecast=0.9167 MASE_Test=0.9167'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M4Comp.py", line 17, in <module>
    tester.run_multiprocessed();
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(nbprocesses)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 94, in getInfo
    logger.info("MODEL_L1 L1_Fit=" + str(self.mFitPerf.mL2) + " L1_Forecast=" + str(self.mForecastPerf.mL2)  + " L1_Test=" + str(self.mTestPerf.mL1) );
Message: 'MODEL_L1 L1_Fit=396.523853843 L1_Forecast=396.523853843 L1_Test=332.833333333'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M4Comp.py", line 17, in <module>
    tester.run_multiprocessed();
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(nbprocesses)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 95, in getInfo
    logger.info("MODEL_L2 L2_Fit=" + str(self.mFitPerf.mL2) + " L2_Forecast=" + str(self.mForecastPerf.mL2)  + " L2_Test=" + str(self.mTestPerf.mL2) );
Message: 'MODEL_L2 L2_Fit=396.523853843 L2_Forecast=396.523853843 L2_Test=396.523853843'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M4Comp.py", line 17, in <module>
    tester.run_multiprocessed();
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(nbprocesses)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 96, in getInfo
    logger.info("MODEL_COMPLEXITY " + str(self.getComplexity()) );
Message: 'MODEL_COMPLEXITY 32'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M4Comp.py", line 17, in <module>
    tester.run_multiprocessed();
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(nbprocesses)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 97, in getInfo
    logger.info("AR_MODEL_DETAIL_START");
Message: 'AR_MODEL_DETAIL_START'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.5/logging/__init__.py", line 983, in emit
    stream.write(msg)
ValueError: I/O operation on closed file.
Call stack:
  File "/usr/local/bin/ipython3", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python3.5/dist-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/IPython/terminal/ipapp.py", line 311, in initialize
    self.init_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 298, in init_code
    self._run_cmd_line_code()
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 423, in _run_cmd_line_code
    self._exec_file(fname, shell_futures=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/shellapp.py", line 353, in _exec_file
    raise_exceptions=True)
  File "/usr/local/lib/python3.5/dist-packages/IPython/core/interactiveshell.py", line 2493, in safe_execfile
    self.compile if kw['shell_futures'] else None)
  File "/usr/local/lib/python3.5/dist-packages/IPython/utils/py3compat.py", line 185, in execfile
    exec(compiler(f.read(), fname, 'exec'), glob, loc)
  File "/home/antoine/dev/python/packages/pyaf/benches/run_benchmark_M4Comp.py", line 17, in <module>
    tester.run_multiprocessed();
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 330, in run_multiprocessed
    pool = mp.Pool(nbprocesses)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 118, in Pool
    context=self.get_context())
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 168, in __init__
    self._repopulate_pool()
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 233, in _repopulate_pool
    w.start()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/lib/python3.5/multiprocessing/popen_fork.py", line 74, in _launch
    code = process_obj._bootstrap()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 57, in run_bench_process
    tester.testSignal(a.mSignal, a.mHorizon)
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 158, in testSignal
    self.trainSignal(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 153, in trainSignal
    self.trainModel(iSignal, iHorizon);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 141, in trainModel
    self.reportModelInfo(lAutoF1);
  File "/home/antoine/dev/python/packages/pyaf/Bench/GenericBenchmark.py", line 132, in reportModelInfo
    iModel.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/ForecastEngine.py", line 40, in getModelInfo
    return  self.mSignalDecomposition.getModelInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/SignalDecomposition.py", line 495, in getModelInfo
    return self.mBestModel.getInfo();
  File "/home/antoine/dev/python/packages/pyaf/TS/TimeSeriesModel.py", line 99, in getInfo
    logger.info("AR_MODEL_DETAIL_END");
Message: 'AR_MODEL_DETAIL_END'
Arguments: ()
       Transformation                                              Model  \
0          _B&IND0817  _B&IND0817_Lag1Trend_residue_zeroCycle_residue...   
1          _B&IND0817  _B&IND0817_Lag1Trend_residue_bestCycle_byL2_re...   
2      Diff_B&IND0817  Diff_B&IND0817_Lag1Trend_residue_zeroCycle_res...   
3      Diff_B&IND0817  Diff_B&IND0817_Lag1Trend_residue_bestCycle_byL...   
4    CumSum_B&IND0817  CumSum_B&IND0817_Lag1Trend_residue_zeroCycle_r...   
5    CumSum_B&IND0817  CumSum_B&IND0817_Lag1Trend_residue_bestCycle_b...   
6      Diff_B&IND0817  Diff_B&IND0817_ConstantTrend_residue_zeroCycle...   
7      Diff_B&IND0817  Diff_B&IND0817_ConstantTrend_residue_bestCycle...   
8   RelDiff_B&IND0817  RelDiff_B&IND0817_ConstantTrend_residue_zeroCy...   
9   RelDiff_B&IND0817  RelDiff_B&IND0817_ConstantTrend_residue_bestCy...   
10  RelDiff_B&IND0817  RelDiff_B&IND0817_LinearTrend_residue_zeroCycl...   
11  RelDiff_B&IND0817  RelDiff_B&IND0817_PolyTrend_residue_zeroCycle_...   
12  RelDiff_B&IND0817  RelDiff_B&IND0817_LinearTrend_residue_bestCycl...   
13  RelDiff_B&IND0817  RelDiff_B&IND0817_PolyTrend_residue_bestCycle_...   
14  RelDiff_B&IND0817  RelDiff_B&IND0817_Lag1Trend_residue_zeroCycle_...   
15  RelDiff_B&IND0817  RelDiff_B&IND0817_Lag1Trend_residue_bestCycle_...   
16     Diff_B&IND0817  Diff_B&IND0817_LinearTrend_residue_zeroCycle_r...   
17     Diff_B&IND0817  Diff_B&IND0817_LinearTrend_residue_bestCycle_b...   
18     Diff_B&IND0817  Diff_B&IND0817_PolyTrend_residue_zeroCycle_res...   
19     Diff_B&IND0817  Diff_B&IND0817_PolyTrend_residue_bestCycle_byL...   
20         _B&IND0817  _B&IND0817_PolyTrend_residue_zeroCycle_residue...   
21         _B&IND0817  _B&IND0817_PolyTrend_residue_bestCycle_byL2_re...   
22         _B&IND0817  _B&IND0817_LinearTrend_residue_zeroCycle_resid...   
23         _B&IND0817  _B&IND0817_LinearTrend_residue_bestCycle_byL2_...   
24         _B&IND0817  _B&IND0817_ConstantTrend_residue_zeroCycle_res...   
25         _B&IND0817  _B&IND0817_ConstantTrend_residue_bestCycle_byL...   
26   CumSum_B&IND0817  CumSum_B&IND0817_PolyTrend_residue_zeroCycle_r...   
27   CumSum_B&IND0817  CumSum_B&IND0817_PolyTrend_residue_bestCycle_b...   
28   CumSum_B&IND0817  CumSum_B&IND0817_LinearTrend_residue_zeroCycle...   
29   CumSum_B&IND0817  CumSum_B&IND0817_LinearTrend_residue_bestCycle...   
30   CumSum_B&IND0817  CumSum_B&IND0817_ConstantTrend_residue_zeroCyc...   
31   CumSum_B&IND0817  CumSum_B&IND0817_ConstantTrend_residue_bestCyc...   

    Complexity  FitCount       FitL1        FitL2  FitMAPE  FitMASE  \
0           32        12  332.833333   396.523854   0.6812   0.9167   
1           40        12  332.833333   396.523854   0.6812   0.9167   
2           64        12  332.833333   396.523854   0.6812   0.9167   
3           72        12  332.833333   396.523854   0.6812   0.9167   
4           64        12  333.666667   397.757673   0.6826   0.9190   
5           72        12  333.666667   397.757673   0.6826   0.9190   
6           32        12  432.055556   521.466846   0.8006   1.1899   
7           40        12  432.055556   521.466846   0.8006   1.1899   
8           32        12  570.333333   646.146784   0.8990   1.5708   
9           40        12  570.333333   646.146784   0.8990   1.5708   
10          48        12  570.333333   646.146784   0.8990   1.5708   
11          48        12  570.333333   646.146784   0.8990   1.5708   
12          56        12  570.333333   646.146784   0.8990   1.5708   
13          56        12  570.333333   646.146784   0.8990   1.5708   
14          64        12  570.333333   646.146784   0.8990   1.5708   
15          72        12  570.333333   646.146784   0.8990   1.5708   
16          48        12  257.435290   319.586531   1.4176   0.7090   
17          56        12  257.435290   319.586531   1.4176   0.7090   
18          48        12  212.960541   255.028115   1.4900   0.5865   
19          56        12  212.960541   255.028115   1.4900   0.5865   
20          16        12  239.659859   291.929380   4.6896   0.6601   
21          24        12  239.659859   291.929380   4.6896   0.6601   
22          16        12  250.075758   298.454838   4.6902   0.6887   
23          24        12  250.075758   298.454838   4.6902   0.6887   
24           0        12  242.611111   303.686607   5.1164   0.6682   
25           8        12  242.611111   303.686607   5.1164   0.6682   
26          48        12  421.142778   542.163819  12.0441   1.1599   
27          56        12  421.142778   542.163819  12.0441   1.1599   
28          48        12  404.888415   576.777791  13.3665   1.1151   
29          56        12  404.888415   576.777791  13.3665   1.1151   
30          32        12  874.847222  1214.810183  30.4514   2.4094   
31          40        12  874.847222  1214.810183  30.4514   2.4094   

    ForecastCount  ForecastL1   ForecastL2  ForecastMAPE  ForecastMASE  \
0              12  332.833333   396.523854        0.6812        0.9167   
1              12  332.833333   396.523854        0.6812        0.9167   
2              12  332.833333   396.523854        0.6812        0.9167   
3              12  332.833333   396.523854        0.6812        0.9167   
4              12  333.666667   397.757673        0.6826        0.9190   
5              12  333.666667   397.757673        0.6826        0.9190   
6              12  432.055556   521.466846        0.8006        1.1899   
7              12  432.055556   521.466846        0.8006        1.1899   
8              12  570.333333   646.146784        0.8990        1.5708   
9              12  570.333333   646.146784        0.8990        1.5708   
10             12  570.333333   646.146784        0.8990        1.5708   
11             12  570.333333   646.146784        0.8990        1.5708   
12             12  570.333333   646.146784        0.8990        1.5708   
13             12  570.333333   646.146784        0.8990        1.5708   
14             12  570.333333   646.146784        0.8990        1.5708   
15             12  570.333333   646.146784        0.8990        1.5708   
16             12  257.435290   319.586531        1.4176        0.7090   
17             12  257.435290   319.586531        1.4176        0.7090   
18             12  212.960541   255.028115        1.4900        0.5865   
19             12  212.960541   255.028115        1.4900        0.5865   
20             12  239.659859   291.929380        4.6896        0.6601   
21             12  239.659859   291.929380        4.6896        0.6601   
22             12  250.075758   298.454838        4.6902        0.6887   
23             12  250.075758   298.454838        4.6902        0.6887   
24             12  242.611111   303.686607        5.1164        0.6682   
25             12  242.611111   303.686607        5.1164        0.6682   
26             12  421.142778   542.163819       12.0441        1.1599   
27             12  421.142778   542.163819       12.0441        1.1599   
28             12  404.888415   576.777791       13.3665        1.1151   
29             12  404.888415   576.777791       13.3665        1.1151   
30             12  874.847222  1214.810183       30.4514        2.4094   
31             12  874.847222  1214.810183       30.4514        2.4094   

    TestCount      TestL1       TestL2  TestMAPE  TestMASE  
0          12  332.833333   396.523854    0.6812    0.9167  
1          12  332.833333   396.523854    0.6812    0.9167  
2          12  332.833333   396.523854    0.6812    0.9167  
3          12  332.833333   396.523854    0.6812    0.9167  
4          12  333.666667   397.757673    0.6826    0.9190  
5          12  333.666667   397.757673    0.6826    0.9190  
6          12  432.055556   521.466846    0.8006    1.1899  
7          12  432.055556   521.466846    0.8006    1.1899  
8          12  570.333333   646.146784    0.8990    1.5708  
9          12  570.333333   646.146784    0.8990    1.5708  
10         12  570.333333   646.146784    0.8990    1.5708  
11         12  570.333333   646.146784    0.8990    1.5708  
12         12  570.333333   646.146784    0.8990    1.5708  
13         12  570.333333   646.146784    0.8990    1.5708  
14         12  570.333333   646.146784    0.8990    1.5708  
15         12  570.333333   646.146784    0.8990    1.5708  
16         12  257.435290   319.586531    1.4176    0.7090  
17         12  257.435290   319.586531    1.4176    0.7090  
18         12  212.960541   255.028115    1.4900    0.5865  
19         12  212.960541   255.028115    1.4900    0.5865  
20         12  239.659859   291.929380    4.6896    0.6601  
21         12  239.659859   291.929380    4.6896    0.6601  
22         12  250.075758   298.454838    4.6902    0.6887  
23         12  250.075758   298.454838    4.6902    0.6887  
24         12  242.611111   303.686607    5.1164    0.6682  
25         12  242.611111   303.686607    5.1164    0.6682  
26         12  421.142778   542.163819   12.0441    1.1599  
27         12  421.142778   542.163819   12.0441    1.1599  
28         12  404.888415   576.777791   13.3665    1.1151  
29         12  404.888415   576.777791   13.3665    1.1151  
30         12  874.847222  1214.810183   30.4514    2.4094  
31         12  874.847222  1214.810183   30.4514    2.4094  
     Transformation                                              Model  \
0        _B&IND0817  _B&IND0817_Lag1Trend_residue_zeroCycle_residue...   
1        _B&IND0817  _B&IND0817_Lag1Trend_residue_bestCycle_byL2_re...   
2    Diff_B&IND0817  Diff_B&IND0817_Lag1Trend_residue_zeroCycle_res...   
3    Diff_B&IND0817  Diff_B&IND0817_Lag1Trend_residue_bestCycle_byL...   
4  CumSum_B&IND0817  CumSum_B&IND0817_Lag1Trend_residue_zeroCycle_r...   

   Complexity  FitCount       FitL1       FitL2  FitMAPE  FitMASE  \
0          32        12  332.833333  396.523854   0.6812   0.9167   
1          40        12  332.833333  396.523854   0.6812   0.9167   
2          64        12  332.833333  396.523854   0.6812   0.9167   
3          72        12  332.833333  396.523854   0.6812   0.9167   
4          64        12  333.666667  397.757673   0.6826   0.9190   

   ForecastCount  ForecastL1  ForecastL2  ForecastMAPE  ForecastMASE  \
0             12  332.833333  396.523854        0.6812        0.9167   
1             12  332.833333  396.523854        0.6812        0.9167   
2             12  332.833333  396.523854        0.6812        0.9167   
3             12  332.833333  396.523854        0.6812        0.9167   
4             12  333.666667  397.757673        0.6826        0.9190   

   TestCount      TestL1      TestL2  TestMAPE  TestMASE  
0         12  332.833333  396.523854    0.6812    0.9167  
1         12  332.833333  396.523854    0.6812    0.9167  
2         12  332.833333  396.523854    0.6812    0.9167  
3         12  332.833333  396.523854    0.6812    0.9167  
4         12  333.666667  397.757673    0.6826    0.9190  
12 0      409.0
1     1122.0
2      213.0
3      359.0
4      348.0
5      452.0
6      599.0
7      254.0
8      187.0
9      562.0
10     387.0
11     215.0
Name: B&IND0817, dtype: float64
12 0     272.0
1     272.0
2     272.0
3     272.0
4     272.0
5     272.0
6     272.0
7     272.0
8     272.0
9     272.0
10    272.0
11    272.0
Name: B&IND0817_Forecast, dtype: float64
FORECAST_DETAIL_ACTUAL B&IND0817 B&IND0817 [  409.  1122.   213.   359.   348.   452.   599.   254.   187.   562.
   387.   215.]
FORECAST_DETAIL_PREDICTED B&IND0817 B&IND0817 [ 272.  272.  272.  272.  272.  272.  272.  272.  272.  272.  272.  272.]
BENCHMARK_PERF_DETAIL B&IND0817 B&IND0817 12 12 5.1549365520477295 Lag1Trend + NoCycle + NoAR 12 0.3648 0.446 0.6608 190.083333333 289.474667861 -0.391774198478
BENCHMARK_SUCCESS 'M4_COMP_BUSINESS-INDUSTRY_B&IND0817_12'
